diagnostic.md

```diff
@@ -93,5 +93,4 @@ var teamMember = {
 
 What is the value of `teamMember['hobby']`?  Of `teamMember['name']`?  Of
 `teamMember['name']['surname']`?
-
-_Replace this text with your answer._
+'exersize', 'Jason', 'Jason' 'Weeks'
```

Comment:
The value of teamMember['name'] is a nested object, { given: 'Jason', surname: 'Weeks' }.  teamMember['name']['surname'] refers to the value of the previous nested object's key "surname", which is "Weeks".